Classes/Private/Manager/Submanagers/OCTSubmanagerCalls.m

```diff
@@ -298,12 +296,24 @@ - (void)toxAV:(OCTToxAV *)toxAV callStateChanged:(OCTToxAVCallState)state friend
 
 - (void)toxAV:(OCTToxAV *)toxAV audioBitRateChanged:(OCTToxAVAudioBitRate)bitrate stable:(BOOL)stable friendNumber:(OCTToxFriendNumber)friendNumber
 {
-    // Lower bitrate if unstable?
+    NSArray *validBitrates = @[@8, @16, @24, @32, @48];
+
+    if (! stable) {
+        NSUInteger currentIndexBitRate = [validBitrates indexOfObject:[NSNumber numberWithInt:bitrate]];
+
+        if (currentIndexBitRate == NSNotFound) {
+            return;
+        }
+
+        NSNumber *newBitrate = [validBitrates objectAtIndex:currentIndexBitRate - 1];
+
+        [self.toxAV setAudioBitRate:newBitrate.intValue force:NO forFriend:friendNumber error:nil];
+    }
```

Comment:
I think we can leave it for now. Currently it is more important just to make things work. I've created an issue so we won't forget about that https://github.com/Antidote-for-Tox/objcTox/issues/53